vips/vip-465/bsctestnet.ts

```diff
@@ -0,0 +1,227 @@
+        signature: "setReserveFactor(uint256)",
+        params: [market.riskParameters.reserveFactor],
+      },
+      {
+        target: market.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, market.initialSupply.amount],
+      },
+      {
+        target: POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            market.vToken.address,
+            market.riskParameters.collateralFactor,
+            market.riskParameters.liquidationThreshold,
+            market.initialSupply.amount,
+            NORMAL_TIMELOCK,
+            market.riskParameters.supplyCap,
+            market.riskParameters.borrowCap,
+          ],
+        ],
+      },
+      {
+        target: market.vToken.address,
+        signature: "transfer(address,uint256)",
+        params: [ethers.constants.AddressZero, parseUnits("0.01692047", 8)], // around $10
+      },
+      (() => {
+        const vTokensMinted = convertAmountToVTokens(market.initialSupply.amount, parseUnits("1", 28));
+        const vTokensRemaining = vTokensMinted.sub(parseUnits("0.01692047", 8));
+        return {
+          target: market.vToken.address,
+          signature: "transfer(address,uint256)",
+          params: [VTREASURY, vTokensRemaining],
+        };
+      })(),
+      {
+        target: market.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, 0],
+      },
+      {
+        target: COMPTROLLER_LIQUID_STAKED_BNB_POOL,
+        signature: "setActionsPaused(address[],uint8[],bool)",
+        params: [[market.vToken.address], [2], true],
+      },
+      {
+        target: market.vToken.address,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [market.riskParameters.protocolSeizeShare],
+      },
+      {
+        target: vankrBNB_LiquidStakedBNB,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [market.riskParameters.protocolSeizeShare],
+      },
+      {
+        target: vBNBx_LiquidStakedBNB,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [market.riskParameters.protocolSeizeShare],
+      },
+      {
+        target: vslisBNB_LiquidStakedBNB,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [market.riskParameters.protocolSeizeShare],
+      },
+      {
+        target: vstkBNB_LiquidStakedBNB,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [market.riskParameters.protocolSeizeShare],
+      },
+      {
+        target: vWBNB_LiquidStakedBNB,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [market.riskParameters.protocolSeizeShare],
+      },
+      {
+        target: COMPTROLLER_LIQUID_STAKED_BNB_POOL,
+        signature: "setLiquidationIncentive(uint256)",
+        params: ["1025000000000000000"],
+      },
```

Comment:
We should add the commands to configure the conversions, on bsctestnet and bscmainnet, with an incentive of 1e14